diff -urNp -x '*.orig' gnome-common-3.18.0.org/macros2/gnome-compiler-flags.m4 gnome-common-3.18.0/macros2/gnome-compiler-flags.m4
--- gnome-common-3.18.0.org/macros2/gnome-compiler-flags.m4	2015-01-27 19:59:24.000000000 +0100
+++ gnome-common-3.18.0/macros2/gnome-compiler-flags.m4	2024-04-22 22:43:26.872914798 +0200
@@ -47,7 +47,6 @@ AU_DEFUN([GNOME_COMPILE_WARNINGS],[
         -Werror=pointer-arith \
         -Werror=init-self \
         -Werror=format-security \
-        -Werror=format=2 \
         -Werror=missing-include-dirs \
         -Werror=return-type \
     "
