--- gnome-common-3.5.91/doc-build/omf.make.old	2012-09-26 19:38:30.541466837 +0200
+++ gnome-common-3.5.91/doc-build/omf.make	2012-09-26 19:38:38.438091893 +0200
@@ -49,7 +49,6 @@ install-data-hook-omf:
 		test -r $$file.out && absfile=$$file.out; \
 		$(INSTALL_DATA) $$absfile $(DESTDIR)$(omf_dest_dir)/$$file; \
 	done
-	-scrollkeeper-update -p $(DESTDIR)$(scrollkeeper_localstate_dir) -o $(DESTDIR)$(omf_dest_dir)
 
 uninstall-local-omf:
 	-for file in $(omffile); do \
@@ -57,7 +56,6 @@ uninstall-local-omf:
 		rm -f $(DESTDIR)$(omf_dest_dir)/$$basefile; \
 	done
 	-rmdir $(DESTDIR)$(omf_dest_dir)
-	-scrollkeeper-update -p $(DESTDIR)$(scrollkeeper_localstate_dir)
 
 clean-local-omf:
 	-for file in $(omffile); do \
