--- gnome-common-2.12.0/doc-build/xmldocs.make.orig	2005-01-03 06:43:23.000000000 +0100
+++ gnome-common-2.12.0/doc-build/xmldocs.make	2006-08-09 19:13:17.384774000 +0200
@@ -34,9 +34,9 @@
 
 
 # **********  Begin of section some packagers may need to modify  **********
-# This variable (docdir) specifies where the documents should be installed.
+# This variable (gtkdochelpdir) specifies where the documents should be installed.
 # This default value should work for most packages.
-docdir = $(datadir)/gnome/help/$(docname)/$(lang)
+gtkdochelpdir = $(datadir)/gnome/help/$(docname)/$(lang)
 
 # **********  You should not have to edit below this line  **********
 xml_files = $(entities) $(docname).xml
@@ -63,15 +63,15 @@
 	fi
 
 install-data-local: omf
-	$(mkinstalldirs) $(DESTDIR)$(docdir)
+	$(mkinstalldirs) $(DESTDIR)$(gtkdochelpdir)
 	for file in $(xml_files); do \
-	  cp $(srcdir)/$$file $(DESTDIR)$(docdir); \
+	  cp $(srcdir)/$$file $(DESTDIR)$(gtkdochelpdir); \
 	done
 	if test "$(figdir)"; then \
-	  $(mkinstalldirs) $(DESTDIR)$(docdir)/$(figdir); \
+	  $(mkinstalldirs) $(DESTDIR)$(gtkdochelpdir)/$(figdir); \
 	  for file in $(srcdir)/$(figdir)/*.png; do \
 	    basefile=`echo $$file | sed -e  's,^.*/,,'`; \
-	    $(INSTALL_DATA) $$file $(DESTDIR)$(docdir)/$(figdir)/$$basefile; \
+	    $(INSTALL_DATA) $$file $(DESTDIR)$(gtkdochelpdir)/$(figdir)/$$basefile; \
 	  done \
 	fi
 
@@ -83,14 +83,14 @@
 	-if test "$(figdir)"; then \
 	  for file in $(srcdir)/$(figdir)/*.png; do \
 	    basefile=`echo $$file | sed -e  's,^.*/,,'`; \
-	    rm -f $(DESTDIR)$(docdir)/$(figdir)/$$basefile; \
+	    rm -f $(DESTDIR)$(gtkdochelpdir)/$(figdir)/$$basefile; \
 	  done; \
-	  rmdir $(DESTDIR)$(docdir)/$(figdir); \
+	  rmdir $(DESTDIR)$(gtkdochelpdir)/$(figdir); \
 	fi
 	-for file in $(xml_files); do \
-	  rm -f $(DESTDIR)$(docdir)/$$file; \
+	  rm -f $(DESTDIR)$(gtkdochelpdir)/$$file; \
 	done
-	-rmdir $(DESTDIR)$(docdir)
+	-rmdir $(DESTDIR)$(gtkdochelpdir)
 
 clean-local: clean-local-doc clean-local-omf
 
--- gnome-common-3.5.91/doc-build/omf.make.orig	2012-09-26 19:39:18.511208144 +0200
+++ gnome-common-3.5.91/doc-build/omf.make	2012-09-26 19:42:35.930138821 +0200
@@ -38,7 +38,7 @@ omf_timestamp: $(omffile)
 	-for file in $(omffile); do \
 	  absfile=$(srcdir)/$$file \
 	  test -r $$file && absfile=$$file \
-	  scrollkeeper-preinstall $(docdir)/$(docname).xml $$absfile $$file.out; \
+	  scrollkeeper-preinstall $(gtkdochelpdir)/$(docname).xml $$absfile $$file.out; \
 	done; \
 	touch omf_timestamp
 
